src/components/redux/books/booksSlice.js

```diff
@@ -1,41 +1,98 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
 /* eslint-disable no-param-reassign */
```

Comment:
**You did great job! Functionality of your app is perfect!**

- [ ] However, we have some cleaning jobs to do! Kindly remove eslint disabler from your file!